providers/openshift/provider.go

```diff
@@ -353,7 +353,10 @@ func (p *OpenShiftProvider) ValidateRequest(req *http.Request) (*providers.Sessi
 	}
 
 	parts := strings.SplitN(auth, " ", 2)
-	token := parts[1]
+	var token string
+	if parts[0] == "Bearer" {
+		token = parts[1]
+	}
 	return &providers.SessionState{User: user.GetName(), Email: user.GetName() + "@cluster.local", AccessToken: token}, nil
```

Comment:
Is it ok here if token is nil, you may have to create providers.SessionState first and then conditionally add a AccessToken key to it only if needed.